Assignment4/drachodran-e22a348/task06.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+"""Task06.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1y93Mcblef-xExVKjumM-Xl1wyT1659mG
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib 

```

Comment:
wont work in python. This is notebook magic

Reply:
Hi Daniel! Could you please tell me more about the problem you see? Do you mean my exercise works on Google colab but it wouldn't execute on a local Python interpreter? As I am new to Python, I can't see the problem

Reply:
line 12 is a magic command (starts with !). Therefore it won't work in Python, it's for the notebooks/bash scripts

Reply:
Dear Daniel, please check that I unmagicked the files. It should be accepted now, I hope.